duckduckgo/Android
# ADS: Report Broken Sites migration

Task/Issue URL: https://app.asana.com/0/0/1203411647299659/f

### Description
Migrate Report Broken Sites screen to new ADS components

### Steps to test this PR

- Install from this branch
- Navigate to a website
- Go to "Report Broken Site" option in the overflow menu

### UI changes
| Before  | After |
| ------ | ----- |
![Screenshot_20221214_100927_DuckDuckGo](https://user-images.githubusercontent.com/20798495/207567541-c38fcdc0-d4d0-4c0e-af76-8196b4c2eded.jpg)|![Screenshot_20221214_101312_DuckDuckGo](https://user-images.githubusercontent.com/20798495/207568133-0e988815-aec5-45ec-9c7f-581fa229d37f.jpg)|
![Screenshot_20221213_155234_DuckDuckGo](https://user-images.githubusercontent.com/20798495/207567691-afec6e7b-dcca-4cbe-8e55-565ebe76d617.jpg)|![Screenshot_20221214_094609_DuckDuckGo](https://user-images.githubusercontent.com/20798495/207567568-e79b8424-9335-4119-9ef9-8ae8870a83aa.jpg)|
